ovirt/resource_ovirt_vm.go

```diff
@@ -938,9 +943,16 @@ func expandOvirtVMDiskAttachment(d interface{}, disk *ovirtsdk4.Disk) (*ovirtsdk
 	if disk != nil {
 		builder.Disk(disk)
 		if v, ok := dmap["size"]; ok {
-			newSize := int64(v.(int)) * int64(math.Pow(2, 30))
-			disk.SetProvisionedSize(newSize)
+			if v != 0 {
+				newSize := int64(v.(int)) * int64(math.Pow(2, 30))
+				disk.SetProvisionedSize(newSize)
+			}
 		}
+	        if v, ok := dmap["alias"]; ok {
+			if v != "" {
+	                	disk.SetAlias(v.(string))
+			}
```

Comment:
Please squash the next commit for formatting with this one.

Reply:
Done